GDJS/Runtime/spriteruntimeobject.ts

```diff
@@ -794,8 +804,14 @@ namespace gdjs {
      * - it's not configured as a loop;
      * - the current frame is the last frame;
      * - the last frame has been displayed long enough.
+     *
+     * @deprecated Use `hasAnimationEnded` instead.
      */
     hasAnimationEnded2(): boolean {
+      return this.hasAnimationEnded();
+    }
+
+    hasAnimationEnded(): boolean {
```

Comment:
I think a good short name is more important than a strict compatibility here because:
- Events compatibility is ensured
- The side effect is a 1-frame difference.
- The method that is reused was already deprecated (I don't know for how long).

It would only sightly impact an old project that is using JS. This is very unlikely and users may not even notice the effect.